src/services/HelpService.js

```diff
@@ -34,6 +36,42 @@ class HelpService {
       JSON.parse(JSON.stringify(createdHelp.ownerId)),
     );
     sendMessage(sendSocketMessageTo, 'new-help', createdHelp);
+
+    this.notificationToFollowers(createdHelp.ownerId, createdHelp.id);
+  }
+
+  async notificationToFollowers(profileId, helpId) {
+    const followers = await this.socialNetworkService.getFollowers(profileId, profileId);
+
+    if (followers) {
+      const ownerTitle = 'Pedido de ajuda criado por uma pessoa que você segue.';
+      const ownerBody = 'Uma das pessoas que você está seguindo, criou uma ajuda.';
+
+      /* eslint-disable no-await-in-loop */
+      for (let i = 0; i < followers.length; i += 1) {
+        const followersNotificationHistory = {
+          userId: followers[i].id,
+          helpId,
+          title: ownerTitle,
+          body: ownerBody,
+          notificationType: notificationTypesEnum.outros,
+        };
+        console.log(followersNotificationHistory);
+        // try {
+        //   await this.NotificationMixin.sendNotification(
+        //     followers.deviceId,
+        //     ownerTitle,
+        //     ownerBody,
+        //   );
+        //   await this.NotificationService.createNotification(
+        //     ownerNotificationHistory,
+        //   );
+        // } catch (err) {
+        //   console.log('Não foi possível enviar a notificação!');
+        //   saveError(err);
+        // }
```

Comment:
Não querendo ser chato mas da pra tirar esse console log e me parece que essa lógica aqui não tá completa, me corrija se estiver errado (além de comentada).

Reply:
Não, pelo o que eu testei ela não está completa, por isso comentei, acho interessante manter com a tag de TODO pois é uma lógica que faz sentido e vou estar abrindo uma issue pra ela ser realizada assim que o PR for aceito, acho legal manter o código comentado para a pessoa que pegar a issue compreender qual o caminho o autor do código estava tentando seguir e entender onde essa lógica deveria ser chamada.